### ISSUE TITLE
Remove "username" field

### ISSUE THREAD

**Issue by pH-7** [pH7Software/pH7-Social-Dating-CMS#609]
Nowadays, usernames aren't as useful and wanted as they used to be.
This field gives a "one more step" to the user in order to join the website, and nothing really more.

This feature should be removed. 
The profile page URL (e.g. `my-dating-website.com/@myusername ` should be replaced to `my-dating-website.com/@USER_ID`

I've already done some work, which should be added to this repo too: https://github.com/pH7Software/pH7Builder/commit/ddc486c8fb870e737e6635cb59173c2109db9397

**Comment by Undefined-Variables** [comment]
I totally disagree, I have been thinking about changing the login page to username or email and allowing logins to be completed with either option. Nobody wants to type out an email address over and over again and usernames are quicker and more convenient. 

I will still require email address for signing up and for communication with users but I think usernames are better for day to day logging in purposes. 

There are a number of big sites still using this approach, Ebay for example!

**Comment by pH-7** [comment]
Thanks for your feedback @Undefined-Variables 
I appreciate it!

**Comment by Undefined-Variables** [comment]
No problem, I just want to say, as you can see I don't get on with Poly but that is no reflection on you! I think you are amazing and I appreciate every minute you have put in to this code. Just wish she/he would stop attacking me at every turn! 

Keep up the good work :)

**Comment by dimuller** [comment]
Hi, I also disagree to remove the username, but I don't use nicknames, I changed the program that the username is generated automatically. It is a perfect identification and can be used to search for a user. Email is not visible for all users, so it can't be used for a search.

**Comment by Undefined-Variables** [comment]
That is an interesting idea, do you have any problems with users trying to bypass the site and pass things like wickr me, facebook or any other way to communicate while avoiding membership fees? 

I only ask because while doing research I have noticed many sites don't allow the passing of personal details, for example if you was to post an email address in a pm the email address is auto removed. Makes me wonder how much of a problem this could be. 

I know the aim is to help people find each other but if they join and use the free trail period to pass contact details the site never creates any revenue.

**Comment by pH-7** [comment]
Thanks for the suggestion.

I reviewed this during backlog cleanup. This request is valid but outside the current 18.x delivery scope, so I’m closing it as not planned for now to keep the tracker focused on active work.

If you want to push this forward, please open a new proposal with a minimal implementation plan (scope, DB/API impact, migration notes, and backward-compatibility expectations).

Take care and thank you for using pH7 Social Dating Builder,

[Pierre-Henry](https://github.com/pH-7)
